app/furniture/marketplace/marketplaces/edit.html.erb

```diff
@@ -1,26 +1,25 @@
 <%- breadcrumb :edit_marketplace, marketplace %>
 
-<%= render "form", marketplace: marketplace %>
+<div class="max-w-2xl self-stretch mx-auto">
```

Comment:
`max-w-prose` instead of `max-w-2xl` might be neat as well (we can play with that in a separate PR tho)